src/irq/irq.go

```diff
@@ -163,8 +163,8 @@ func applyIRQConfig(
 	}
 
 	// Range over IRQ tuning array
-	for i, irqTuning := range config.Data.Interrupts {
-		log.Printf("\nRule #%d ( %s )", i+1, formatIRQRule(irqTuning))
+	for label, irqTuning := range config.Data.Interrupts {
+		log.Printf("\nRule #%s ( %s )", label, formatIRQRule(irqTuning))
```

Comment:
Given the labels, this should now print:
Rule: a random cpu

Instead of:
Rule #a random cpu ( CPUs: 18, scaling_governor: performance, min_freq: 0.6GHz, max_freq: 5.6GHz )